tests/README.md

```diff
@@ -201,23 +256,26 @@ def test_display_historical(mocker):
 ```
 
 Important notes to go through:
-1. Use `@pytest.mark.record_stdout`to capture `sdtout`.
+
+1. Use `@pytest.mark.record_stdout` to capture `sdtout`.
    1. Recorded stdout is saved as fixtures in a txt file
    2. If your view function only plots a chart, and does not output anything in `sdtout`, you can skip unit testing your `view.py`
-   3. Make sure to mock `visualize_output` method.
+   3. Make sure to mock `visualize_output` whenever possible.
```

Comment:
@tehcoderer @montezdesousa does this still applies? We still have some references to `visualize_output` across our codebase.

Reply:
Only if it's a matplotlib chart aka 'portfolio/po'

'OpenBBFigure' has logic not to open pywry window in tests